### PR TITLE
Add readmes for SSD300/SSD512 models trained on VOC07+12

### PR DESCRIPTION
**Pull request by vanyalzr**

### CODE REVIEW

**Comment by vanyalzr** [comment]
@AlexKoff88

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
16:20:03  ssd512-int8-onnx-0001 not in index.md
Please add link to model in index.md

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
check failed : ssd512-int8-sparse-v2-onnx-0001 not in index.md file

**Comment by kchechil** [comment]
Jenkins: test this please

**Comment by eaidova** [comment]
@vanyalzr, please move  added readme files from `intel_models/<model_name>` to `models/intel/<model_name>`

File: intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection |
+| GFLOPs            | 62.815 |
+| MParams           | 26.285 |
+| Source framework  | PyTorch    |
+
+## Accuracy
+
+The quality metric (mAP) calculated on VOC07+12 validation dataset has the value of 78.28%.
```
Comment (eaidova) [inline intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md]: I think you should not specify metric value twice, the table will be enough.
Also, possibly, it makes sense to add information about mAP definition

File: intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## Input
+
+Image, shape - `1,3,300,300`, format is `B,C,H,W` where:
+
+- `B` - batch size
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `RGB`
```
Comment (eaidova) [inline intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md]: model converted with `--reverse_input_channels` flag, so Channel order is `BGR`

File: intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# ssd300-fp32-onnx-0001
```
Comment (eizamaliev) [inline intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md]: Add `Legal information` part (see other model's description for example).

File: intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+The model output for `ssd300-fp32-onnx-0001` is the usual object detection output for the 21 different classes present in the VOC database.
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection |
+| GFLOPs            | 62.815 |
+| MParams           | 26.285 |
+| Source framework  | PyTorch    |
```
Comment (eizamaliev) [inline intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md]:
```suggestion
| Source framework  | PyTorch\*    |
```

File: intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md (hunk 1 above)
Comment (vanyalzr) [reply intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md]: Done.

File: intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md (hunk 2 above)
Comment (vanyalzr) [reply intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md]: Done.

File: intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md (hunk 3 above)
Comment (vanyalzr) [reply intel_models/ssd300-fp32-onnx-0001/description/ssd300-fp32-onnx-0001.md]: Done.

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: models/intel/ssd512-int8-sparse-v2-onnx-0001/description/ssd512-int8-sparse-v2-onnx-0001.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# ssd512-int8-sparse-v2-onnx-0001
```
Comment (IRDonch) [inline models/intel/ssd512-int8-sparse-v2-onnx-0001/description/ssd512-int8-sparse-v2-onnx-0001.md]: Please update `tools/downloader/license.txt` with the new models.

**Comment by AlexKoff88** [comment]
All models are postponed to the next release. Don't merge!

**Pull request closed without merging**